src/libslic3r/GCode.cpp

```diff
@@ -355,7 +355,7 @@ bool GCode::gcode_label_objects = true;
                 new_filament_e_feedrate = new_filament_e_feedrate == 0 ? 100 : new_filament_e_feedrate;
 
                 config.set_key_value("max_layer_z", new ConfigOptionFloat(gcodegen.m_max_layer_z));
-                config.set_key_value("relative_e_axis", new ConfigOptionBool(RELATIVE_E_AXIS));
+                //config.set_key_value("relative_e_axis", new ConfigOptionBool(m_config->use_relative_e_distances));
```

Comment:
I suggest to get rid of unused code. Or make a comment with explanation why this code is commented and in which cases it can be useful(may be for debug purpose)

Reply:
Keep it with a comment is a good idea.

Reply:
I deleted it because it referred to a variable that do not exist anymore (RELATIVE_E_AXIS).

Reply:
Noted